.github/workflows/crawl.yml

```diff
@@ -0,0 +1,101 @@
+name: Crawl
+
+on: workflow_call
+
+jobs:
+  crawl:
+    runs-on: ubuntu-latest
+    env:
+      # this is used in a number of steps, and is set here to avoid
+      # duplication.
+      RHSM_OFFLINE_TOKEN: ${{ secrets.RHSM_OFFLINE_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS_KERNEL_CACHE }}'
+      - uses: google-github-actions/setup-gcloud@v1
+
+      - uses: ./.github/actions/env
+
+      - name: Run crawl
+        env:
+          REDHAT_USERNAME: ${{ secrets.REDHAT_USERNAME }}
+          REDHAT_PASSWORD: ${{ secrets.REDHAT_PASSWORD }}
+          REDHAT_SUBSCRIPTION_ORG_ID: ${{ secrets.REDHAT_SUBSCRIPTION_ORG_ID }}
+          REDHAT_SUBSCRIPTION_ACTIVATION_KEY: ${{ secrets.REDHAT_SUBSCRIPTION_ACTIVATION_KEY }}
+          SUSE_MIRRORING_USERNAME: ${{ secrets.SUSE_MIRRORING_USERNAME }}
+          SUSE_MIRRORING_PASSWORD: ${{ secrets.SUSE_MIRRORING_PASSWORD }}
+          UBUNTU_ESM_SUBSCRIPTION_TOKEN: ${{ secrets.UBUNTU_ESM_SUBSCRIPTION_TOKEN }}
+          UBUNTU_FIPS_SUBSCRIPTION_TOKEN: ${{ secrets.UBUNTU_FIPS_SUBSCRIPTION_TOKEN }}
+        run: |
+          if ! make -k crawl 2> >(tee /tmp/make-crawl-stderr >&2) ; then
+              touch /tmp/crawl-failed
+          fi
+
+          if [[ -f /tmp/crawl-failed ]] || grep -Eq '\*\*\* \[[a-zA-Z0-9-]+\] Error' /tmp/make-crawl-stderr ; then
+              echo >&2 "'make crawl' failed. See the output of the 'Crawl package repositories' step in the crawl job for further details."
```

Comment:
Think this comment is outdated, `Crawl package repositories` was the name it had in CircleCI. Maybe we should update it?

Reply:
It almost feels entirely redundant beyond "crawling failed" because everything is logged in line now

Reply:
Yeah, that wasn't the case when we originally implemented this and behavior has changed at some point in time. The idea was to have this check on a separate job, so that repackaging can happen even if some repo was not successfully crawled, we might want to go back to this behavior.